javascript-timers/interval.js

```diff
@@ -0,0 +1,24 @@
+/* var countdown = 4;
+setInterval(() => {
+  if (countdown <= 4 && countdown >= 1) {
+    var $countdownDisplay = document.querySelector('.countdown-display');
+    $countdownDisplay.textContent = countdown;
+    countdown--;
+  }
+  if (countdown === 0) {
+    $countdownDisplay.textContent = '~Earth Beeeelooowwwww Us~';
+    clearInterval();
+  }
+}, 1000); */
+
+setInterval(changeCountdown, 1000);
+var countdown = 4;
+function changeCountdown() {
+  var $countdownDisplay = document.querySelector('.countdown-display');
+  $countdownDisplay.textContent = countdown;
+  countdown--;
+  if (countdown < 0) {
+    $countdownDisplay.textContent = '~Earth below Us~';
+    clearInterval(1);
```

Comment:
You can't be confident that `1` is the id for this interval, extensions can create intervals as well. You need to save the return value of calling `setInterval` so you know the exact id for this exact interval.